### ISSUE TITLE
what did the accuracy mean when use ctc_bath_cost?

### ISSUE THREAD

**Issue by foye501** [foye501/hello-world#2]
m = Bidirectional(GRU(rnnunit,return_sequences=True,implementation=2),name='blstm2')(m)
y_pred = Dense(nclass,name='blstm2_out',activation='softmax')(m)

basemodel = Model(inputs=input,outputs=y_pred)
basemodel.summary()

labels = Input(name='the_labels',shape=[maxlabellength],dtype='float32')
input_length = Input(name='input_length', shape=[1], dtype='int64')
label_length = Input(name='label_length', shape=[1], dtype='int64')

loss_out = Lambda(ctc_lambda_func, output_shape=(1,), name='ctc')([y_pred, labels, input_length, label_length]) 

model = Model(inputs=[input, labels, input_length, label_length], outputs=loss_out)

adam = Adam()
#tf_adam = tf.train.AdamOptimizer ()
#adadelta = Adadelta()
model.compile(loss={'ctc': lambda y_true, y_pred: y_pred}, optimizer=adam,metrics=['accuracy'])

[https://github.com/xiaomaxiao/keras_ocr/blob/master/CRNN/crnn.ipynb](url)

How did the accuracy come, what used to computed it,thanks in advance
